usn/ubuntu_server_netboot.py

```diff
@@ -131,8 +131,10 @@ def add_kernel_params(self, params, install_only=False):
                     replace = "--- %s" % param_str
                 line = line.replace("---", replace)
             new_cfg += "%s\n" % line
+        # Remove trailing newlines from file 
+        new_cfg = new_cfg.rstrip()
         self.cfg = new_cfg
-
+                   
```

Comment:
And this one

Reply:
This change isn't present on the final solution. The timeout feature required the fixing of the whitespace in the expected config along with the  two routines. The two issues are related and therefore in my opinion should be in the same commit. I can squash the two commits into one for the sake of time. Otherwise I leave it to you to make the final changes. I hope I've added value to your tool. Thank you.

Reply:
No problem - I think we might just be miscommunicating. I've gone ahead and made the changes I was trying to highlight and merged (removing arbitrary whitespace changes in the code itself, unrelated to the whitespace fix you contributed). I also noticed an optimization with re.sub we could make, so I've added that as a subsequent commit.

Thank you for your patience, and thank you for this feature @toledo-systems !